pro/src/ui-kit/form/SirenInput/SirenInput.tsx

```diff
@@ -7,9 +7,7 @@ import { useField } from 'formik'
 const formatSiren = (value: string) => {
   // remove character when when it's not a number
   // this way we're sure that this field only accept number
```

Comment:
Je trouverai cela plus compréhensible comme ça :

```suggestion
const formatSiren = (value: string) => {
  const lastChar = [...value].reverse()(0)

  // remove character when it's not a number
  // this way we're sure that this field only accept number
  if (lastChar && isNaN(Number(value))) {
    return value.slice(0, -1)
  }
  return humanizeSiren(value)
}
```